### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by lollypop27**
Title of the PT story: Add Translations

Link to PT story: https://www.pivotaltracker.com/story/show/129743625

Description of the PR:

1. update locales files with translations
2. add translations to templates that are part of the app

Ready for review!
@tochman @diraulo

### CODE REVIEW

**Comment by lollypop27** [comment]
@tochman and @diraulo will fix fist thing

**Comment by lollypop27** [comment]
@tochman @diraulo fixed conficts

**Comment by lollypop27** [comment]
@tochman @diraulo I have fixed error.

**Comment by tochman** [comment]
Are we happy with this team? Ping @lollypop27 @MikaelFeher @luciademoja @diraulo

File: app/views/devise/passwords/new.html.haml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 = form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f|
   = devise_error_messages!
   .field
-    = f.label :email
+    = label_tag(t(:email))
```
Comment (tochman) [inline app/views/devise/passwords/new.html.haml]: Hmm... why the change to `label_tag` ?

File: features/step_definitions/steps.rb [hunk 2]
```diff
@@ -89,4 +97,28 @@
...
+Then(/^the locale is set to "([^"]*)"$/) do |locale|
+  I18n.locale = locale
+end
+
+Then(/^I should see :"([^"]*)"$/) do |text|
+  page.has_content? I18n.t(text)
+end
+
+Then(/^I click :"([^"]*)" button$/) do |button|
+  click_button?(I18n.t(button))
```
Comment (tochman) [inline features/step_definitions/steps.rb]: Plz omit the `?`

File: features/step_definitions/time_steps.rb [hunk 3]
```diff
@@ -12,4 +12,8 @@
 
 After do
   Timecop.return
+end
+
+Given(/^the application is set to "([^"]*)"$/) do |arg|
+  I18n.locale = :sv
```
Comment (diraulo) [inline features/step_definitions/time_steps.rb]: shouldn't this be set to whatever the value of `arg` is?

Comment (tochman) [inline features/step_definitions/time_steps.rb]: yes, of course.

**Comment by diraulo** [comment]
@tochman Is this good to go now? Also note that we had to disable turbolinks here because it wasn't playing nice with our JS functions.

**Comment by tochman** [comment]
Oh. I would like to see that in action. Hold of merging this for a moment.

File: app/views/devise/unlocks/new.html.haml [hunk 4]
```diff
@@ -2,7 +2,7 @@
 = form_for(resource, as: resource_name, url: unlock_path(resource_name), html: { method: :post }) do |f|
   = devise_error_messages!
   .field
-    = f.label :email
+    = label_tag(t(:email))
```
Comment (tochman) [inline app/views/devise/unlocks/new.html.haml]: Needs fixing

**Comment by tochman** [comment]
@diraulo I enabled turbolinks again and I go green across the board.

**Comment by diraulo** [comment]
Okay.. Some of those didn't pass until we disabled turbolinks.. Unless I overlooked something

**Comment by tochman** [comment]
Alright. IMO this is ready to merge.

**Comment by diraulo** [comment]
Alright let's :shipit:
